index.html

```diff
@@ -21,7 +21,11 @@
   </head>
   <body>
     <header>
-      <nav class="navbar navbar-custom">
+      <div class="notice-bar hide">
+  <div class="text-center notice-text">November release is out! <a href="">Read about the new features and fixes.</a></div>
```

Comment:
This URL should link to the latest release, I guess.

Reply:
XD totally forgot about it